BuildAndCopy.cmd

```diff
@@ -25,14 +25,25 @@ echo ** Additional Build Parameters:%AdditionalBuildCommand%
 echo.
 :: Build MSBuild
 call "%~dp0build.cmd" /t:Rebuild %AdditionalBuildCommand%
+set BUILDERRORLEVEL=%ERRORLEVEL%
 
 :: Kill Roslyn, which may have handles open to files we want
 taskkill /F /IM vbcscompiler.exe
 
+if %BUILDERRORLEVEL% NEQ 0 (
+    echo.
+    echo Failed to build with errorlevel %BUILDERRORLEVEL% 1>&2
```

Comment:
I added the log path to the last message from `build.cmd` (the common script).  That work?